verify/verify/src/lib.rs

```diff
@@ -1,5 +1,5 @@
 #![deny(warnings, rust_2018_idioms)]
-#![cfg_attr(test, feature(avx512_target_feature, abi_vectorcall))]
+#![cfg_attr(test, feature(avx512_target_feature, abi_vectorcall, asm))]
```

Comment:
Why is `asm` required here ?

Reply:
If no `asm` feature, error occurred, when
```
cd verify/verify
RUSTFLAGS="-C target-feature=+avx2" cargo test --release
```
![image](https://user-images.githubusercontent.com/30716372/63849371-842faa00-c9c4-11e9-8bca-e36a69f5c8d3.png)

Reply:
`assert_instr` is using `asm` feature: https://github.com/rust-lang/stdarch/blob/1d1ab19f8123a5d8a9a7031446fd63629f2484b4/crates/assert-instr-macro/src/lib.rs#L143